README.md

```diff
@@ -137,14 +136,14 @@ npm test
 
 ## 👥 Authors <a name="authors"></a>
 
-👤 **Gino Anderson Canales**
-
-- GitHub: [@anderCM](https://github.com/anderCM)
-
 👤 **Jose Montoya**
 
 - GitHub: [@jmonto55](https://github.com/jmonto55)
 
+👤 **Nelson Sakwa**
+
+- Behance: [@jmonto55](https://www.behance.net/sakwadesignstudio)
+
```

Comment:
- [Optional] Great Job for adding the author to the design :+1: , however, this `author` section was prepared only for those who contributed to this repo, so add it to the "Acknowledgements" section [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) :+1:

Reply:
Thank you, @binhussen